### ISSUE TITLE
Fixed ignored ChronicleMethodReaderTests

### ISSUE THREAD

**Issue by RobAustin** [OpenHFT/Chronicle-Queue#660]
Fix ignored test
see https://teamcity.chronicle.software/viewLog.html?buildId=403827&buildTypeId=OpenHFT_ChronicleQueue4_Snapshot
```
ChronicleMethodReaderTest.shouldReturnNoMoreThanTheSpecifiedNumberOfMaxRecords
ChronicleMethodReaderTest.shouldFilterByExclusionRegex
ChronicleMethodReaderTest.shouldNotRewindPastStartOfQueueWhenDisplayingHistory
ChronicleMethodReaderTest.shouldConvertEntriesToText
```

**Comment by RobAustin** [comment]
@peter-lawrey asked for these test to be ignored so added @Ignore("https://github.com/OpenHFT/Chronicle-Queue/issues/660") to them

**Comment by hft-team-city** [comment]
Released in [Chronicle-Queue-5.20.23](https://github.com/OpenHFT/Chronicle-Queue/releases/chronicle-queue-5.20.23), [BOM-2.20.61](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.61)
